tests/dataset_histograms/histogram_error_estimator_test.py

```diff
@@ -121,6 +139,17 @@ def test_get_ratio_dropped_linf(self, linf_bound, expected):
         self.assertAlmostEqual(estimator.get_ratio_dropped_linf(linf_bound),
                                expected)
 
+    @parameterized.parameters((0, 1), (0.5, 0.89), (1, 0.78), (2, 0.76),
+                              (40, 0))
+    # there 1 is contribution 40 and 10 contribution 1.
+    # total contribution = 1*40+10*1 = 50
+    # when linf_bound = 0.5, left after contribution bounding 11*0.5=5.5, i.e.
```

Comment:
how linf can be a double and not an integer?

Reply:
linf_bound is max contribution per partition, which means

`max_contributions_per_partition` for COUNT

`max_sum_per_partition` for SUM (which can be double)